features/admin/resetting_password.feature

```diff
@@ -0,0 +1,35 @@
+@administrator_login
+Feature: Resetting an administrator's password
+    In order to login to my administrator account when I forget my password
+    As an not logged in Administrator
+    I need to be able to reset my password
+
+    Background:
+        Given there is an administrator "sylius@example.com" identified by "sylius"
+
+    @todo
+    Scenario: Resetting an administrator password
```

Comment:
it isn't resetting the password but only sending an email with the next steps.
probably you should change this scenario name